capstone-frontend/src/components/UserCard.jsx

```diff
@@ -77,7 +104,9 @@ export class UserCard extends Component {
       <Col className='user-card' xs={{ span: '2' }} >
         <img id='user-profile' src={this.props.user.profileImageUrl} alt='Profile' />
         <div> {this.props.user.fullName} </div>
-        <Button className='mt-2' variant='primary' onClick={this.addFriend}> Add Friend </Button>
+        {this.state.isFriend 
+          ? <Button className='mt-2' variant='danger' onClick={this.removeFriend}> Remove Friend </Button>
+          : <Button className='mt-2' variant='primary' onClick={this.addFriend}> Add Friend </Button> }
```

Comment:
Could you make these buttons the same width? I think it would make it more clean and consistent

Reply:
Sure! I'll set their widths to be a fixed % of the box they're in. Here's what it looks like now:

![Screenshot 2020-07-27 at 9 37 24 AM](https://user-images.githubusercontent.com/43327083/88548503-2dc05680-cfed-11ea-92c1-738628eddb55.png)